AtlasStarter/Program.cs

```diff
@@ -0,0 +1,185 @@
+            // the bottom of this file that serves as a "mapping class" -- the 
+            // driver maps the C# class to the BSON stored in MongoDB.
+
+            // Using mapping classes is strongly advised, but if you
+            // don't create them, you can always use the more generic BsonDocument
+            // type.
+
+            try
+            {
+                client = new MongoClient(mongoUri);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return;
+            }
+
+            // Provide the name of the database and collection you want to use.
+            // If they don't already exist, the driver and Atlas will create them
+            // automatically when you first write data.
+            var dbName = "testDatabase";
+            var collectionName = "testCollection";
+
+            try
+            {
+                collection = client.GetDatabase(dbName)
+                    .GetCollection<Recipe>(collectionName);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return;
+            }
+
+            /*      *** INSERT DOCUMENTS ***
+             * 
+             * You can insert individual documents using collection.Insert(). 
+             * In this example, we're going to create 4 documents and then 
+             * insert them all in one call with InsertMany().
+             */
+
+            var docs = Recipe.GetRecipes();
+
+            try
+            {
+                collection.InsertMany(docs);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return;
+            }
+
+            /*      *** FIND DOCUMENTS ***
+             * 
+             * Now that we have data in Atlas, we can read it. To retrieve all of
+             * the data in a collection, we call Find() with an empty filter. 
+             * The Builders class is very helpful when building complex 
+             * filters, and is used here to show its most basic use.
+             */
+
+            var allDocs = collection.Find(Builders<Recipe>.Filter.Empty)
+                .ToList();
+
+            foreach (Recipe doc in allDocs)
+            {
+                // Do something with each document ...
+            }
+
+            // We can also find a single document. Let's find the first document
+            // that has the string "potato" in the Ingredients list. Again we
+            // use the Builders class to create the filter, and a LINQ
+            // statement to define the property and value we're after:
+
+            var findFilter = Builders<Recipe>
+                .Filter.AnyEq(t => t.Ingredients,
+                "potato");
+
+            var findResult = collection.Find(findFilter).FirstOrDefault();
+
+            if (findResult == null)
+            {
+                return;
```

Comment:
I think it would be helpful for end users if we printed a message in both the "found a document" and "didn't find a document" cases so they can see that something happened.

Reply:
+1 on the printing out results in either case!

We also might not want a `return` here.  If the user's playing around with this script, they might change it so no results are found at this juncture.  In this case, there's still no reason the script shouldn't be allowed to continue onward.